examples/question_answering/pyproject.toml

```diff
@@ -6,10 +6,18 @@ authors = ["Kolena Engineering <eng@kolena.io>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.9,<3.10"
 kolena = ">=0.82.0,<1"
 s3fs = "^2022.7.1"
+torch = [
+  {markers = "sys_platform == 'darwin' and platform_machine == 'arm64'", url = "https://download.pytorch.org/whl/cpu/torch-2.0.1-cp39-none-macosx_11_0_arm64.whl"},
+  {markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'", url = "https://download.pytorch.org/whl/cpu/torch-2.0.1-cp39-none-macosx_10_9_x86_64.whl"},
+  {markers = "sys_platform == 'linux' ", url = "https://download.pytorch.org/whl/cpu/torch-2.0.1%2Bcpu-cp39-cp39-linux_x86_64.whl"},
+  {markers = "sys_platform == 'win32' ", url = "https://download.pytorch.org/whl/cpu/torch-2.0.1%2Bcpu-cp39-cp39-win_amd64.whl"},
+]
```

Comment:
What's the behavior if you don't have this block? `poetry add torch` seems to work on my mac, is this failing on other system?

Reply:
I'll give that a try. I believe we want to force it to only use the CPU version of torch though